backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/log2lin.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+import numpy as np
+
+from nodes.properties.inputs import BoolInput, ImageInput, SliderInput
+from nodes.properties.outputs import ImageOutput
+
+from .. import adjustments_group
+
+
+@adjustments_group.register(
+    schema_id="chainner:image:log2lin",
+    description="Convert all channels in an image to a scene linear encoding using the Cineon logarthmic function set.",
```

Comment:
Typo and could you explain this a bit more? I have no idea what this is for from just reading the description. (Descriptions are markdown, so you can also link to websites to do the explaining for you.)

```suggestion
    description="Convert all channels in an image to a scene linear encoding using the Cineon logarithmic function set.",
```